spark/v4.0/spark/src/test/java/org/apache/iceberg/spark/TestSparkSessionCatalog.java

```diff
@@ -98,4 +104,42 @@ public void testLoadFunction() {
 
     // TODO: fix loading Iceberg built-in functions in SessionCatalog
   }
+
+  @Test
+  public void testCreateViewIfNotExistsWithExistingHiveView() {
+    String viewName = "default.existing_hive_view";
+
+    resetSparkCatalog();
+    try {
+      // create Hive view
+      sql("CREATE VIEW %s AS SELECT 1 AS id", viewName);
+    } finally {
+      setUpCatalog();
+    }
+
+    try {
+      sql("CREATE VIEW IF NOT EXISTS %s AS SELECT 2 AS id", viewName);
```

Comment:
Does this exercise `SparkSessionCatalog#createView(ViewInfo)`?  In my environment, CREATE VIEW is planned as V1 `CreateViewCommand` (not the V2 `ViewCatalog` path), so the tests may pass without hitting the code change.

Reply:
Could we also verify the test exercises the V2 view path so it actually hits `SparkSessionCatalog#createView(ViewInfo)`?